docker/structs.go

```diff
@@ -0,0 +1,8 @@
+package docker
+
+import "github.com/docker/engine-api/types"
+
+// VolumeRequest is taken from
+// https://github.com/docker/engine-api/types/types.go
+type VolumeRequest types.VolumeCreateRequest
```

Comment:
hmmm, the more I read docker remote volume plugin code the more it looks like that none of the volume RPC structures are public.

For instance looks like the plugin RPC structures are generated using the [plugin-rpc tool] (https://github.com/docker/docker/blob/28dde09cdf2a48dcdb6ce665cd2b912175dfb871/pkg/plugins/pluginrpc-gen/README.md) that works on a [private interface](https://github.com/docker/docker/blob/3d13fddd2bc4d679f0eaa68b0be877e5a816ad53/volume/drivers/extpoint.go#L30) which generates code like [this] (https://github.com/docker/docker/blob/3d13fddd2bc4d679f0eaa68b0be877e5a816ad53/volume/drivers/proxy.go)

Not sure if it's even a good idea to use `types.VolumeCreateRequest` unless I am still missing something. We may need to keep the approach that we have. 

@erikh @yuva29 wdyt?